lib/rpc/server/peer.go

```diff
@@ -426,3 +434,15 @@ type peer struct {
 	// doneCh is the channel that is closed when this peer shuts down
 	doneCh chan struct{}
 }
+
+func isDeadlineExceededError(err error) bool {
```

Comment:
Nit: Would utils/errors.go be a better place for this?

Reply:
It is pretty specific in the implementation - not really something I'd use elsewhere.